client/src/app/utils/utils.ts

```diff
@@ -94,10 +94,10 @@ export const getValidatedFromError = (error: unknown | undefined) => {
 };
 
 export const standardURLRegex =
-  /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/gi;
+  /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/;
```

Comment:
So that's it?! I see the ignore case flag is redundant.  The global flag removal also makes sense (apart from the back reference craziness) since you want the full string to be a valid url.

WDYT about adding a "^" to the beginning and a "$" to the end to make sure the whole string is exactly a URL?

Reply:
Just updated with your suggestions.